client.go

```diff
@@ -364,6 +364,28 @@ func (c *Client) SetTagByLocator(locator string, tags *Tags) (*Tags, error) {
 	return tags, nil
 }
 
+func (c *Client) DownloadBuildLog(buildId int, w io.WriteCloser) error {
+	url := c.host + fmt.Sprintf("/downloadBuildLog.html?buildId=%d", buildId)
+	var body io.Reader
```

Comment:
From the top-most example of [`net/http`'s Overview](https://golang.org/pkg/net/http/#pkg-overview), it seems we don't need this variable, and can pass nil in place of `body` at L371.

Reply:
Done